core.coffee

```diff
@@ -230,7 +230,7 @@
 			apiPrefix: '/'
 			passthrough: {}
 			passthroughByMethod: {}
-			constructor: (params) ->
+			constructor: (params, backendParams) ->
```

Comment:
Yeah, the `backendParams` are for the backend, this is just to show the expected constructor signature